### PR TITLE
tests/nested/manual/fde-on-classic: retry occassionally failing check

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**

### CODE REVIEW

File: tests/nested/manual/fde-on-classic/task.yaml [hunk 1]
```diff
@@ -126,8 +126,8 @@ execute: |
   # Test aborting a kernel installation before a reboot
   refresh_rebooting_snap pc-kernel-new.snap pc-kernel no-reboot
   remote.exec sudo snap abort --last=install
-  # Make sure everything is in the same state
-  not remote.exec stat /run/mnt/ubuntu-boot/EFI/ubuntu/try-kernel.efi
+  # Make sure everything is in the same state (might need a retry)
```
Comment (bboozzoo) [inline tests/nested/manual/fde-on-classic/task.yaml:129]: maybe this should `snap watch --last=install` for the change to complete before checking?

Comment (alfonsosanchezbeato) [reply tests/nested/manual/fde-on-classic/task.yaml:129]: Isn't that wait already happenning in the call to `snap abort`? Or for that command that is not really the case?

**Review by Meulengracht — APPROVED** [review]

File: tests/nested/manual/fde-on-classic/task.yaml [hunk 2]
```diff
@@ -126,6 +126,7 @@ execute: |
   # Test aborting a kernel installation before a reboot
   refresh_rebooting_snap pc-kernel-new.snap pc-kernel no-reboot
   remote.exec sudo snap abort --last=install
+  remote.exec sudo snap watch --last=install
```
Comment (bboozzoo) [inline tests/nested/manual/fde-on-classic/task.yaml:129]: just checked that snap watch exits with a error if the change was canceled (and possibly on failure too), so this should likely be:

```suggestion
  remote.exec sudo snap watch --last=install || true
```

Comment (alfonsosanchezbeato) [reply tests/nested/manual/fde-on-classic/task.yaml]: ok, let's try again, thanks

**Review by bboozzoo — APPROVED** [review]
LGTM

**Pull request merged**